README.md

```diff
@@ -6,6 +6,11 @@ go-ipfix is an IPFIX library that can be used to implement an IPFIX exporter, wh
 ## Try it out
 This IPFIX library can be used to build an exporter. Please check out the [exporter tests](https://github.com/vmware/go-ipfix/blob/main/pkg/exporter/process_test.go) to get an idea on how to build exporter on top of TCP and UDP transport protocols given a IPFIX collector.
 
+To deploy a standalone go-ipfix collector, which is used to decode and log the IPFIX records, you can use:
```

Comment:
Better to have a sub-section like "Deploy Go-IPFIX Collector"

Reply:
Will do. Thanks!